src/main/java/com/getjavaobj/training/algo/init/gzabarte/TaskCh02N0013Test.java

```diff
@@ -0,0 +1,24 @@
+package com.getjavaobj.training.algo.init.gzabarte;
+
+import com.getjavaobj.training.algo.init.gzabarte.TaskCh02N0013;
+import org.junit.Assert;
```

Comment:
You need to create and use your own Assert class. Read about it in the file with tasks in the Testing paragraph